src/Logic.hs

```diff
@@ -394,6 +394,22 @@ handleEventInternal triggerConfig mergeWindowExemption event = case event of
   PushPerformed branch sha        -> handleTargetChanged branch sha
   Synchronize                     -> synchronizeState
 
+handlePullRequestOpenedByUser
+  :: forall es. (Action :> es, RetrieveEnvironment :> es)
+  => TriggerConfiguration
+  -> MergeWindowExemptionConfiguration
+  -> PullRequestId
+  -> Branch
+  -> BaseBranch
+  -> Sha
+  -> Text
+  -> Username
+  -> Text
+  -> ProjectState
+  -> Eff es ProjectState
+handlePullRequestOpenedByUser triggerConfig mergeWindowExemption pr branch baseBranch sha title author body state =
```

Comment:
I don't think you need to create a whole new function for this. This would be the only usesite of `handlePullRequestOpened`. 

Instead you can insert the pull request into the project's state and handle whatever was in the description, if there was anything, in a single function.

Reply:
I created this separate function because it is not the only usesite of `handlePullRequestOpened`. It is also used in `clearPullRequest`: 
https://github.com/channable/hoff/blob/72511a0afe696f85d64003164de926d7c33a0f1b/src/Logic.hs#L362-L373

Reply:
Ah missed that. My mistake